cmd/utils.go

```diff
@@ -88,3 +89,19 @@ func limitCount(objs, total int) string {
 
 	return fmt.Sprintf(" (%d of %d)\n", objs, total)
 }
+
+// checkErrors handles specific behavior for certain errors,
+// such as printing an warning for the API mismatch error
+func checkErrors(c *client.Client, err error) error {
```

Comment:
small nit, but mind if we call this `checkAPICompatibility` again? The old name made it more clear that this is the main intention of `checkErrors`. If we need to do more error checking in the future then we can call it `checkErrors`, imo :)